### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mobileqa**
This change is [Reviewable](https://reviewable.io/reviews/mobileqa/pull_request_test/2)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/mobileqa/pull_request_test/pull/2" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # pull_request_test
```
Comment (mobileqa) [inline README.md]: 1

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
...
 .
```
Comment (mobileqa) [inline README.md]: 2

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
+2
```
Comment (mobileqa) [inline README.md]: 3

**Pull request closed without merging**